### PR TITLE
installer: git clone using ssh URL if https fails

### PR DESCRIPTION
**Pull request by pierrickrouxel**
I added IBM i support. Just few changes.

### CODE REVIEW

**Comment by dabou** [comment]
oh yes, oh-my-zsh on IBM i !
Thank's :tada:

File: lib/theme-and-appearance.zsh [hunk 1]
```diff
@@ -17,6 +17,8 @@ then
     # coreutils, so prefer it to "gls".
     gls --color -d . &>/dev/null 2>&1 && alias ls='gls --color=tty'
     colorls -G -d . &>/dev/null 2>&1 && alias ls='colorls -G'
+  elif [[ "$(uname -s)" == "OS400" ]]; then
```
Comment (apjanke) [inline lib/theme-and-appearance.zsh]: Could this be done with a generic test of whether `ls -G` is supported, instead of adding an OS-specific test?

**Comment by pierrickrouxel** [comment]
Ok I will do that.

**Comment by apjanke** [comment]
See also #4988, which looks like it's attempting the same thing.

**Comment by pierrickrouxel** [comment]
I removed changes on ls appearance. The #4988 solves the problem more efficiently.

**Comment by mcornella** [comment]
The HTTPS-to-SSH remote URL change won't be needed now that #5169 introduces the ability to change the remote URL via the `REMOTE` variable (so you could run `REMOTE=git@github.com:robbyrussell/oh-my-zsh.git sh install.sh`, for example). This PR will be automatically closed when #5169 is merged.

**Comment by mcornella** [comment]
As I said above, #5169 adds the ability to configure the remote URL, so you can now use an SSH remote. 

If you want to help speed up the testing and merge (which should happen early next week), you can wget or curl it from the URL `https://raw.githubusercontent.com/mcornella/oh-my-zsh/refactor-installer/tools/install.sh`. Read the comments at the top of the script to know how to call it.

Thanks!

**Comment by mcornella** [comment]
Closed by #5169.

**Pull request closed without merging**